Changelog.md

```diff
@@ -8,6 +8,7 @@ Language Features:
 
 Compiler Features:
  * Analysis: Properly detect circular references to the bytecode of other contracts across all function calls.
+ * Antlr Grammar: Fix parsing of import paths involving properly distinguishing between empty and non-empty string literals in general.
```

Comment:
This is not a compiler feature. I think it could classify as documentation update? Or tools?

Reply:
Yeah, not sure... we can also drop it from the changelog :-). Or actually I could just call it a bugfix...

Reply:
Or we do introduce a new category to the changelog - I don't see any precedent for an entry like this actually...

Reply:
Made it a Bugfix now, but also fine with introducing a new category or dropping the entry.